README.md

```diff
@@ -2,34 +2,30 @@
 This repo contains various documents related to network observability (a.k.a. netobserv):
 
 ### Repositories
-* [network-observability-operator](https://github.com/netobserv/network-observability-operator)
+* [netobserv-operator](https://github.com/netobserv/network-observability-operator)
     OpenShift / Kubernetes operator for network observability.
     This operator will deloy the following components:
-    * [goflow2-kube-enricher](https://github.com/netobserv/goflow2-kube-enricher)
-        This component implements the following libraries:
-        * [loki-client-go](https://github.com/netobserv/loki-client-go)
-        An HTTP client to send logs to Loki server
-        * [goflow2-loki-exporter](https://github.com/netobserv/goflow2-loki-exporter)
-        A Loki exporter plugin
-    * [network-observability-console-plugin](https://github.com/netobserv/network-observability-console-plugin)
-    Network Observability plugin for the Openshift Console
-    * [nflow-generator](https://github.com/netobserv/nflow-generator)
-    A fake legacy v5 netflow generator
+    * [eBPF Agent](https://github.com/netobserv/netobserv-ebpf-agent): An eBPF agent that captures and exports network flows.
+    * [flowlogs-pipeline](https://github.com/netobserv/flowlogs-pipeline): A configurable flow collector, enricher and metrics producer.
+    * [netobserv-plugin](https://github.com/netobserv/network-observability-console-plugin):
+    The NetObserv plugin for the Openshift Console.
+* [nflow-generator](https://github.com/netobserv/nflow-generator):
+A fake legacy v5 netflow generator
+* etc.
 
 ### Get started
-The easiest way to get started is to use the [Network Observability Operator](https://github.com/netobserv/network-observability-operator) that will deploy all the components for you.
+The easiest way to get started is to use the [NetObserv Operator](https://github.com/netobserv/network-observability-operator) that will deploy all the components for you. It is available via [OLM](https://operatorhub.io/operator/netobserv-operator).
```

Comment:
Do we also plan to rename github repos ?
https://github.com/netobserv/network-observability-operator

Reply:
https://github.com/netobserv/network-observability-console-plugin

Reply:
I fear that would be too much hassle... (and same for quay)

Reply:
Seems that github has redirect capabilities for both pages / git commands:
https://docs.github.com/en/repositories/creating-and-managing-repositories/renaming-a-repository

I agree we should not do that if it impacts too much things behind the scene but we may give a try for consistency